addon/services/unified-event-handler.js

```diff
@@ -56,8 +61,10 @@ export default Ember.Service.extend(Ember.Evented, {
    * @return {Void}
    */
   register(target, eventName, callback) {
-    let handlerInfo = this._registerDOMHandler(target, eventName);
-    this._registerEmberHandler(handlerInfo, callback);
+    if (!this.get('isFastBoot')) {
```

Comment:
Both here and below, can we use a return-early pattern instead?

```js
if (this.get('isFastBoot')) {
  return;
}
```

It's easier to parse the flow of the logic IMO.

Reply:
No problem, will update.